usagov/usagov-2021
# usagov-1262-Twitter-CEO: Update lookup.js

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1262

## Description

Changed the javascript response from "Twitter" to "X"


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- On CEO page fill out the form. On the results page make sure Twitter is changed to X.
![Screenshot 2023-12-11 at 3 43 23 PM](https://github.com/usagov/usagov-2021/assets/108532577/ce31187f-ce53-445c-a7d0-b1248b7d18cd)

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
